HyperPlay-Gaming/hyperplay-desktop-client
# [FIX] Protocol Handler duplicated calls

This PR should fix the intermittent issue of the Protocol handler being called twice from the store even though only one click event was emitted from the Play Game button.

## To test
- Use the artifact from the build, dev mode won't work.
- Install the app
- Go to the store and launch anything from there.
- Game should launch only once
- If not installed, the dialog asking to install should appear only once.

---

Use the following Checklist if you have changed something on the Backend or Frontend:
